City-of-Helsinki/hel-fi-drupal-grants
# fix: AU-1964: Fixing dependency error

# [AU-1964](https://helsinkisolutionoffice.atlassian.net/browse/AU-1964)

## What was done
This pull request fixes the following error when performing a fresh Drupal-install: 

`The service "grants_handler.application_handler" has a dependency on a non-existent service "grants_attachments.attachment_handler`

## How to install
Make sure your instance is up and running on the correct branch.
  * `git checkout feature/AU-1964-dependency-fix`
  * `make fresh`
  * `make drush-cr`

## How to test
Run `drush si -y && drush cr && drush si --existing-config -y && drush cim -y && drush gwi --force` in your local development environment and check that the install goes through to the end.

[AU-1964]: https://helsinkisolutionoffice.atlassian.net/browse/AU-1964?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
